File: SEP3-TIER2/src/Server/Model/PendingInvitation.java
package Server.Model;

public class PendingInvitation {

    private int projectid;
    private String username;

    public PendingInvitation(int projectId, String username) {
        this.projectid = projectId;
        this.username = username;
    }

    public PendingInvitation() {
    }

    public int getProjectId() {
        return projectid;
    }

    public void setProjectId(int projectId) {
        this.projectid = projectId;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    @Override
    public String toString() {
        return "PendingInvitation{" +
                "projectId=" + projectid +
                ", username='" + username + '\'' +
                '}';
    }
}

File: SEP3-TIER1/BlazorTest/Model/PendingInvitation.cs
using Newtonsoft.Json;

namespace BlazorTest.Model
{
    public class PendingInvitation
    {
        [JsonProperty("projectid", NullValueHandling = NullValueHandling.Ignore)]
        public int ProjectId { get; set; }

        [JsonProperty("username", NullValueHandling = NullValueHandling.Ignore)]
        public string Username { get; set; }
    }
}
